frontend/src/state/reducers/scheduleReducer.ts

```diff
@@ -268,6 +269,10 @@ export const scheduleReducer = (
         draft.present.warnings = namedSchedule.warnings;
         return draft;
       }
+      case getType(resetScheduleAction): {
+        draft = initialState;
+        return draft;
+      }
     }
```

Comment:
Another thing, you can actually reuse the resetUserAction here instead of making a separate one for Schedule (you can have the ScheduleAction track UserActions)